src/main/resources/com/adaptavist/tm4j/jenkins/extensions/configuration/Tm4jGlobalConfiguration/config.jelly

```diff
@@ -23,6 +23,8 @@
                         </f:entry>
                     </f:radioBlock>
 
+                    <f:entry>
```

Comment:
@pelizza this adds and extra space between components.

![image](https://user-images.githubusercontent.com/359978/56989925-0b315a80-6b6a-11e9-9d34-6d453bf75765.png)

I couldn't come up with a better solution as it seems that that's the way jelly `<f:validateButton>` template shows error messages. I can try to create some custom validation button if you think it's worth of it.

Reply:
That's fine Otavio, this space will help a bit already.